chore(TypeScript): Change comments in #9100 to single asterisks

ScottSWu opened the pull request:
Double asterisk comments are parsed as JSDoc comments, which causes
problems with the closure compiler. This commit changes them all to single asterisks.

princemaple commented:
Looks like they have been removed in this **already merged** PR https://github.com/angular/angular/pull/9148

ScottSWu committed:
chore(TypeScript): Change comments in #9100 to single asterisks

Double asterisk comments are parsed as JSDoc comments, which causes
problems with the closure compiler.

vicb commented:
Do you want to rebase ? If not please close
